src/webviews/extension-side/dataviewer/dataViewerCommandRegistry.ts

```diff
@@ -99,7 +99,10 @@ export class DataViewerCommandRegistry implements IExtensionSingleActivationServ
                         this.debugService.activeDebugSession.configuration
                     );
 
-                    pythonEnv && (await this.dataViewerDependencyService.checkAndInstallMissingDependencies(pythonEnv));
+                    pythonEnv &&
+                        (await this.dataViewerDependencyService.checkAndInstallMissingDependenciesOnEnvironment(
+                            pythonEnv
+                        ));
                 }
```

Comment:
wait, will this work with remote debugging.
i don't think so.
ie if im debugging a remote kernel, then will this fixed path work?
i think we'll need to get that working as well for the web. 
in there loader it didn't work, but now that we're enabling df viewer for web, 100% of web users will use remote kernels, hence this won't work when they're debugging. 

again, i think this is what @IanMatthewHuff mentioned.
we'd need to change the code as we should be sending the request using the debugger instead of interpreter or kernel (whether it's local or remote)

Reply:
I don't think the command to debug a python file with the Python extension is working on the web. I can't get to make it work.

Besides that, this approach you are pointing out would be broken on remote today, is that the case?

I would like to separate the problem of the debugger into a separate PR, because the debugger in itself is fairly complex and it does not seem to be working working on the web anyway.

Reply:
Also, this file is using the pre-existing `dataViewerDependencyService`, not the new one